---a/net/minecraft/world/IInteractionObject.java
+++b/net/minecraft/world/IInteractionObject.java
@@ -4,10 +4,10 @@
 import net.minecraft.entity.player.InventoryPlayer;
 import net.minecraft.inventory.Container;
 
-public interface IInteractionObject extends IWorldNameable
-{
+public interface IInteractionObject extends IWorldNameable {
+
     Container createContainer(InventoryPlayer playerInventory, EntityPlayer playerIn);
 
     String getGuiID();
-}
+}
